cpp/tensorrt_llm/batch_manager/runtimeBuffers.cpp

```diff
@@ -792,6 +792,7 @@ void RuntimeBuffers::setFromInputs(RequestVector const& contextRequests, Request
     }
     if (modelConfig.usePromptTuning())
     {
+        promptTuningBuffers->runtimeIsChunkedContext = runtimeIsChunkedContext;
```

Comment:
same here. initialize during ctor because this is more like a static attribute throughout the lifetime